src/extension.ts

```diff
@@ -30,11 +30,15 @@ export function activate(context: ExtensionContext) {
   context.subscriptions.push(diagnostics, statusBar);
 
   workspace.onDidChangeTextDocument(e => {
-    diagnostics.delete(e.document.uri);
-    showStatus(e.document.uri.fsPath);
+    if (e) {

```

Comment:
👍  The type definition says it's not `| undefined` but who knows if at runtime that it actually isn't sometimes `undefined`

Reply:
I had some stacktraces in dev console, so I made sure all of them check `e` before use